index.bs

```diff
@@ -145,6 +145,9 @@ cache whenever the user clears their cookies or when session cookies expire.
 Note: A site can clear the browser's `Accept-CH` cache for its origin by sending an empty `Accept-CH`
 header in a response. This sets the origin's [=/client hints set=] to an empty set.
 
+Note: There MUST only be one `Accept-CH` header per-request and if more than one is present the browser
```

Comment:
"per response"? I guess that's what confused me..

I wonder if we'd event know, as e.g. intermediaries can coalesce multiple headers into a single comme-delimited one. I'm not sure our network stack doesn't do the same.

Reply:
We collapse them right now, but an empty one causes a drop of the cache whether or not any others are set. If a cache drop wasn't an empty line it would be easier to just say we combine them.

Reply:
In that case, it may be best to only consider a single empty header as a cache-dropper?